src/log/file/log_file_daemon.cc

```diff
@@ -8,6 +8,8 @@
 
 #include "squid.h"
 
+#include "compat/socket.h"
+#include "compat/unistd.h"
```

Comment:
With this the redundant `unistd.h` include below can be removed.

Also, please place an empty line between local and system include lists.
```suggestion
#include "compat/unistd.h"

```